src/entry.js

```diff
@@ -1,9 +1,11 @@
 import { h, render } from 'preact';
 
+import * as AppImport from './index';
+
 let root = document.getElementById('app') || document.body.firstElementChild;
 
 const init = async () => {
-	const { default: App } = await import('./index');
+	const { default: App } = AppImport;
```

Comment:
Why was this change needed? 👀

Reply:
Same here

Reply:
Please check the PR description for error logs